### ISSUE TITLE
Cannot find module 'postcss-custom-properties'

### ISSUE THREAD

**Issue by michaelb-01** [angular/material2#8725]
#### Bug, feature request, or proposal:
I'm not sure what exactly causes this, but i've just got the following error:

`Cannot find module 'postcss-custom-properties'`

When I do npm install postcss-custom-properties, I then get:

`Cannot find module '@ngtools/webpack'`

#### What are the steps to reproduce?
Providing a StackBlitz/Plunker (or similar) is the *best* way to get the team to see your issue. <br/>
Plunker starter (using on `@master`): https://goo.gl/uDmqyY<br/>
StackBlitz starter (using latest `npm` release): https://goo.gl/wwnhMV<br/>

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 5 
Angular material 5.0.0-rc.2

**Comment by CharlBest** [comment]
This is probably not what is wrong with yours but I also had this problem and this fixed it for me. 
So my scenario:
I created a new Angular CLI project (ng new). Not sure if it had something to do with it but I'm using SASS (--style=scss). Then I ejected the webpack config (webpack.config.json) with "ng eject" and then I uninstalled Angular CLI te remove the dependency from my project. I then tried to "npm start" which uses webpack by it failed with error: Cannot find module 'postcss-custom-properties'. 

To solve this:
I had to have Angular CLI installed even though I'm only using webpack and all of its plugins. So "npm install @angular/cli". You probably have it installed already so I'm not sure sorry. I just thought that this could help the devs here.
